src/Client.php

```diff
@@ -65,6 +65,8 @@ public function query() {
     /**
      * Executes the given Query and returns the result
      *
+     * This method should not be called manually.
```

Comment:
I got the syntax from this very diff :-) (https://github.com/twingly/twingly-search-api-php/pull/11/files#diff-4528a13e17ddcea8ae7919377e58843bL33)